.travis.yml

```diff
@@ -4,6 +4,7 @@ install:
 - pip install tox
 script:
 - tox
+- tox -e functional
```

Comment:
Not required. Tox by default will run anything in the envlist, so you just need to add it to 

https://github.com/sc68cal/Kostyor-cli/blob/master/tox.ini#L2